### DISCUSSION TITLE
No gross charges for MS DRG codes creating "missing property" error

### DISCUSSION THREAD

**Discussion by MS2606** [CMSgov/hospital-price-transparency#99] [Q&A] [answered]
while running the hospital MRF on the validator tool, it shows an error of missing property for the "gross_charges" and "discounted_cash". There are no charges and discounted cash for MS DRGs. When referring the provided CMS JSON schema example (V2.0.0_JSON_Format_Example) on github, 

  "standard_charge_information": [
    {
      "description": "Major hip and knee joint replacement or reattachment of lower extremity without mcc",
      "code_information": [
        {
          "code": "470",
          "type": "MS-DRG"
        },
        {
          "code": "175869",
          "type": "LOCAL"
        }
      ],
      "standard_charges": [
        {
          "minimum": 20000,
          "maximum": 20000,
          "setting": "inpatient",
          "payers_information": [
            {
              "payer_name": "Platform Health Insurance",
              "plan_name": "PPO",
              "standard_charge_dollar": 20000,
              "standard_charge_algorithm": "MS-DRG",
              "estimated_amount": 22243.34,
              "methodology": "case rate"
            }

This example does not include "gross_charges" and "discounted_cash" for the MS DRG as well. Similar error happening for maximum/minimum columns as well. 

1. Could you provide an explanation of how gross_charges and discounted_cash should be encoded for MS DRGs?
2. For the CDM codes there is no payer rate and therefore no minimum/maximum, so how maximum/minimum should be coded for the CDM codes?

**Comment by RidenourK** [comment, marked as answer]
@MS2606. Thank you for your question. 

As indicated in [the JSON documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/JSON), the minimum, maximum, gross_charge, and discounted_cash attributes are not required attributes if you have no applicable data to encode.  Leaving these attributes blank should not produce errors when running an MRF through the Online Validator if the MRF adheres to the technical specifications, including the conditional requirements, found in the Hospital Price Transparency Data Dictionary GitHub Repository. Please refer the [JSON examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples/JSON) for examples of how to encode these attributes. 

CMS strongly recommends hospitals start by downloading one of the template layouts or schema to create the machine-readable file, as opposed to converting an existing file.
